src/couch/src/couch_debug.erl

```diff
@@ -281,18 +285,20 @@ help(Unknown) ->
     io:format("    ---~n", []),
     ok.
 
--spec busy(ProcessList :: [process_name()], Threshold :: pos_integer()) ->
+-spec busy([process_name()] | pid(), Threshold :: pos_integer()) ->
```

Comment:
This is not the intend. The intend is to make the following spec possible.

```
-spec busy([process_name() | pid()], Threshold :: pos_integer()) ->
    [Process :: process_name() | pid()].
```

And to make it clearer and allow re-usability of a type we can define it as.

```
-type process_name() :: atom().
-type process() :: process_name() | pid().
....

-spec busy([process()], Threshold :: pos_integer()) ->
    [Process :: process()].
```